src/Elevators/Elevators.csproj

```diff
@@ -18,4 +18,10 @@
     <PackageReference Include="Tailwind.Extensions.AspNetCore" Version="1.0.0" />
   </ItemGroup>
 
+  <ItemGroup>
```

Comment:
proč je tohle přidané do csproj?

Reply:
Pravděpodobně jsem to tam vložil, když jsem něco zkoušel. Odstraním to